.pre-commit-config.yaml

```diff
@@ -9,3 +9,9 @@ repos:
     rev: 22.10.0
     hooks:
     -   id: black
+-   repo: https://github.com/pycqa/isort
```

Comment:
Can you directly use ruff?

Reply:
yes, thanks! I did not know ruff. That is great.
I hope I use it correctly now.